### PR TITLE
Upgrade roumen/feed to 2.9.8 for #1192

### PR DESCRIPTION
**Pull request by phecho**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
I'll do this on the 2.0 branch. Thanks though. :)

File: composer.lock [hunk 1]
```diff
@@ -538,7 +538,7 @@
             },
             "dist": {
                 "type": "zip",
-                "url": "https://api.github.com/repos/doctrine/cache/zipball/47c7128262da274f590ae6f86eb137a7a64e82af",
+                "url": "http://packagist.phpcomposer.com/files/doctrine/cache/47c7128262da274f590ae6f86eb137a7a64e82af.zip",
```
Comment (GrahamCampbell) [inline composer.lock]: all these changes here weren't want we wanted too

Comment (GrahamCampbell) [inline composer.lock]: also, i'd certainly recommend against this on security grounds anyway - you don't want your dependences intercepted on download and replaced with injected bad code

Comment (phecho) [inline composer.lock]: @GrahamCampbell 
OK, Thank you.

**Pull request closed without merging**